larky/src/main/java/com/verygood/security/larky/modules/vgs/aus/spi/AccountUpdaterService.java

```diff
@@ -0,0 +1,34 @@
+package com.verygood.security.larky.modules.vgs.aus.spi;
+
+import lombok.Builder;
+import lombok.Data;
+
+public interface AccountUpdaterService {
+  /**
+   * Get updated info for the provided card
+   *
+   * @param number card's number
+   * @param expireMonth card's expiration month
+   * @param expireYear card's expiration year
+   * @param name the name on the card
+   * @param clientId client id of service account to access calm API
+   * @param clientSecret client secret of service account to access calm API
+   * @return the updated card
+   */
+  Card lookupCard(
```

Comment:
Not sure of the point of this method since it returns what it was passed in

Reply:
Or does it return a _different_ card? 🤔

Reply:
A card could be expired, exp date changed, or stolen, so the number changed. It's still the same card, but a different number or exp date. So this service and this method is for looking up the given card number and see if there's updates for the card. The customer would like to use the new card number is there's one in case of the original one is expired.